src/Altinn.Profile/Models/NotificationAddressModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Altinn.Profile.Validators;
+
+namespace Altinn.Profile.Models
+{
+    /// <summary>
+    /// Represents a notification address
+    /// </summary>
+    public class NotificationAddressModel
+    {
+        /// <summary>
+        /// <see cref="NotificationAddressID"/>
+        /// </summary>
+        public int NotificationAddressID { get; set; }
+
+        /// <summary>
+        /// Country code for phone number
+        /// </summary>
+        [CustomRegexForNotificationAddresses("CountryCode")]
+        public string CountryCode { get; set; }
+
+        /// <summary>
+        /// Email address
+        /// </summary>
+        [CustomRegexForNotificationAddresses("Email")]
+        public string Email { get; set; }
+
+        /// <summary>
+        /// Phone number
+        /// </summary>
+        public string Phone { get; set; }
```

Comment:
Should this be included?
```suggestion
        [CustomRegexForNotificationAddresses("Phone")]
        public string Phone { get; set; }
```